### ISSUE TITLE
sitewide 508/ADA review

### ISSUE THREAD

**Issue by mollieru** [partnerandpartners/alliance-for-youth-action#63]
I'm noticing a few places where our colors aren't passing 508 standards for visual accessibility.  I think this is OK for launch, but might be worth thinking how to make sure all links and colors are accessible to all levels of visibility, as well as making sure the site works w screen readers. 

A few places where our colors aren't passing right now:
![screen shot 2017-04-04 at 2 24 49 pm](https://cloud.githubusercontent.com/assets/3680494/24672424/a93cd37a-1942-11e7-826a-d07f7f038e8c.png)
![screen shot 2017-04-04 at 2 24 34 pm](https://cloud.githubusercontent.com/assets/3680494/24672420/a59f23a8-1942-11e7-91d8-d14c79ea6b18.png)

A handy tool for helping w color contrast checking (and general 508 resources): http://webaim.org/resources/contrastchecker/

**Comment by gregmihalko** [comment]
Right, @mollieru this is a good point - we can try to adjust some of the places where the contrast is not ideal, but the earlier feedback from the team was that they wanted links to stand out a bit more with color, so I'm wondering what we can do given their selections? Any thoughts? 

Also, we have gone through some tab/screen reader tests and it looks like we're okay there.
